front/src/main/scss/main.scss

```diff
@@ -32,14 +39,14 @@
   pointer-events: none;
 }
 
-.subpipe:after {
+.subpipe:after{
```

Comment:
please re-run beautify from IDE, as there are a few styling changes like this one that are incorrect & unneeded